shared-libs/user-management/src/users.js

```diff
@@ -780,11 +776,11 @@ module.exports = {
     return Promise
       .all([
         getAllUsers(),
-        getAllUserSettings(),
-        getFacilities()
+        getAllUserSettings()
       ])
-      .then(([ users, settings, facilities ]) => {
-        return mapUsers(users, settings, facilities);
+      .then(([ users, settings ]) => {
+        return facility.list(users, settings)
+          .then(facilities => mapUsers(users, settings, facilities));
```

Comment:
I really dislike double tiered `then`s. Can we refactor this to use async/await?